bevy_asset_loader_derive/src/lib.rs

```diff
@@ -455,30 +466,82 @@ fn parse_field(field: &Field) -> Result<AssetField, Vec<ParseFieldError>> {
+                                                        .get_ident()
+                                                        .unwrap()
+                                                        .clone();
+                                                    if path == SamplerAttribute::FILTER {
+                                                        if let Expr::Path(ExprPath {
+                                                            path, ..
+                                                        }) = &named_value.value
+                                                        {
+                                                            let filter_result =
+                                                                FilterType::try_from(
+                                                                    path.get_ident()
+                                                                        .unwrap()
+                                                                        .to_string(),
+                                                                );
 
-                                            if let Ok(sampler) = sampler_result {
-                                                builder.sampler = Some(sampler);
-                                            } else {
-                                                errors.push(ParseFieldError::UnknownAttribute(
-                                                    named_value.value.into_token_stream(),
-                                                ));
+                                                            if let Ok(filter) = filter_result {
+                                                                builder.filter = Some(filter);
+                                                            } else {
+                                                                errors.push(ParseFieldError::UnknownAttribute(
+                                                                    named_value.value.clone().into_token_stream(),
+                                                                ));
+                                                            }
+                                                        } else {
+                                                            errors.push(
+                                                                ParseFieldError::WrongAttributeType(
+                                                                    named_value.into_token_stream(),
+                                                                    "path",
+                                                                ),
+                                                            );
+                                                        }
+                                                    }
+                                                }
+                                                Meta::Path(path) => {
```

Comment:
This currently allows the configuration of both `clamp` and `repeat` on the same asset. Would it maybe be better to use a named value as it's done for the filter?